### PR TITLE
server: add diff score check for balance.

### PR DESCRIPTION
**Pull request by qiuyesuifeng**

### CODE REVIEW

**Comment by shenli** [comment]
LGTM

File: server/balancer.go [hunk 1]
```diff
@@ -82,6 +84,14 @@ func (cb *capacityBalancer) checkScore(cluster *clusterInfo, oldPeer *metapb.Pee
 		return false
 	}
 
+	// If the diff score is in defaultScoreFraction range, then we will do nothing.
+	diffScore := oldStoreScore - newStoreScore
```
Comment (siddontang) [inline server/balancer.go]: diffScore can be negative?

**Comment by siddontang** [comment]
LGTM

File: server/balancer.go (hunk 1 above)
Comment (qiuyesuifeng) [inline server/balancer.go]: No, it has been checked above.

**Pull request merged**